README.md

```diff
@@ -35,6 +35,14 @@ UPPHUB_BEGIN
       "status": "stable",
       "category": "scientific",
       "readme": "https://raw.githubusercontent.com/mirek-fidler/gdal/master/README.md"
+    },
+    { "name": "MessageCtrl",
+      "packages": [ "MessageCtrl"],
+      "description": "A passive notifications widget for U++",
+      "repository": "https://github.com/ismail-yilmaz/MessageCtrl.git",
+      "status": "stable",
+      "category": "widget",
+      "readme": "https://github.com/ismail-yilmaz/MessageCtrl/blob/main/README.md"
```

Comment:
We have main branch instead of master. I am curios how TheIDE will behave...

@mirek-fidler are we read for non master branch...

Reply:
@klugier 

Well, that's due to political correctness policy of GitHub. It automatically created `main`, instead of `master` branch.

Reply:
No, this will happened more often we need to make sure that both branches are supported. I would like to hear Mireks opinion, he probably have wider perspective. Maybe there is a magic "branch" parameter or something like that.

Reply:
Ok, I'd changed it to 'master', but now it is reverted to `main` again, so that Mirek can comment on it.